class-13.md

```diff
@@ -0,0 +1,138 @@
+# Intro to Blockchain Development
+
+### Class 13 - More Hyperledger Examples 
+
+We got started with private blockchains last week, exploring them through Hyperledger Composer and Fabric. Now we'll go a bit more in depth and look onto some specific code examples further.
+
+
+
+Reading & Videos:
+
+- [Composer Sample Networks](https://github.com/hyperledger/composer-sample-networks)* - This is a set of composer examples from the hyperledger organization. Try running one to see how it works, and get some ideas for how to implement solutions to problems in a permissioned blockchain.
+- [Fabric Insurance App](https://github.com/IBM/build-blockchain-insurance-app) - We'll look briefly at a full Hyperledger Fabric application, instead of just using the composer framework on top of it. This is a more in-depth example, showing how an insurance network, asset owners and the police could all interact in an automated claims process on the blockchain.
+
+### More Composer Examples
+
+Businesses have addressed a multitude of problems in a supply chain setup with software solutions in recent years, and blockchain can improve upon them. Let's take a look first at an identity example:
+
+**[Composer Identity Example](https://github.com/hyperledger/composer-sample-networks/tree/master/packages/pii-network)** - This shows how two users can add/revoke access to shared personally identifiable information.
+
+### Docker
+
+Before running our first fabric example, we'll run through some basic docker commands, go over what docker is and how to use it in a few basic configurations.
+
+Docker is just a lightweight VM that can be easily reproduced and shared, making it simple to reproduce environments between a developer's computer and a production server. It is also very useful for making each peer in a permissioned blockchain have a predictable environment for chaincode.
+
```

Comment:
```
Before running our first fabric example, we'll run through some basic docker commands, go over what docker is and how to use it in a few basic configurations.

Docker is just a lightweight VM that can be easily reproduced and shared, making it simple to reproduce environments between a developer's computer and a production server. It is also very useful for making each peer in a permissioned blockchain have a predictable environment for chaincode.
```

Before running our first fabric example, we'll introduce Docker: a lightweight VM that simplifies reproducing environments between a developer's computer and a production server. It is also very useful for making each peer in a permissioned blockchain have a predictable environment for chaincode.

We'll run through some basic docker commands and configurations.